cwms/locations/physical_locations.py

```diff
@@ -4,9 +4,36 @@
 from pandas import DataFrame
 
 import cwms.api as api
+import cwms.catalog.catalog as ct
 from cwms.types import Data
 
 
+def get_locations_catalog(page: Optional[str] = None,
+                          page_size: Optional[int] = 5000,
+                          unit_system: Optional[str] = None,
+                          office: Optional[str] = None,
```

Comment:
also lets have the cwms-python use office_id.  I know that CDA goes back and forth. but for consistency lets have the parameters for cwms-python all be office_id.